scripts/convert-dlrs-rules.apex

```diff
@@ -34,6 +30,37 @@ for (dlrs__LookupRollupSummary2__mdt dlrsRule : dlrs__LookupRollupSummary2__mdt.
       operation = dlrsRule.dlrs__AggregateOperation__c;
     }
   }
+   if(invalidChildren.contains(dlrsRule.dlrs__ChildObject__c)){
+    //we will debug a list of unmigrateable rules to assist with the creation of the flow actions
+    Map<String,String> unmigratableRule = new Map<String,String>();
+    if(dlrsRule.dlrs__RelationshipCriteria__c != null){
+      unmigratableRule.put('SOQL Where Clause To Exclude Calc Items',dlrsRule.dlrs__RelationshipCriteria__c);
+    }
+    unmigratableRule.put('Records to rollup','Same as Prior records to rollup');
+    unmigratableRule.put('Prior records to rollup','provide the collection of child records (assuming rollup not from parent record else collection of parent records)');
```

Comment:
This description for records to rollup and prior records to rollup is incorrect. Your description for `Prior records to rollup` belongs instead with `Records to rollup`. The description for `Prior records to rollup` should be something like: "A collection variable with `{!$Record__Prior}` in it, when using after update or after create and update flows".

Reply:
thanks will fix